Builds fails for 0.2.0 , 0.1.2.0

vvdec 0.1.0.0 builds just fine.
Both latest tags failed with both cmake and regular make.
[0.2.0.txt](https://github.com/fraunhoferhhi/vvdec/files/5662002/0.2.0.txt)
[0.1.2.txt](https://github.com/fraunhoferhhi/vvdec/files/5662003/0.1.2.txt)

What could be an issue?

```cpp
/home/zen/Vault/Downloads/vvdec-0.2.0.0/source/Lib/CommonLib/UnitTools.cpp: In function ‘bool PU::getInterMergeSubPuMvpCand(const PredictionUnit&, MergeCtx&, bool&, int)’:
/home/zen/Vault/Downloads/vvdec-0.2.0.0/source/Lib/CommonLib/UnitTools.cpp:3088:43: error: ‘void* memset(void*, int, size_t)’ clearing an object of non-trivial type ‘class Mv’; use assignment or value-initialization instead [-Werror=class-memaccess]
 3088 |         memset( mi.mv, 0, sizeof( mi.mv ) );
      |                                           ^
In file included from /home/zen/Vault/Downloads/vvdec-0.2.0.0/source/Lib/CommonLib/Unit.h:56,
                 from /home/zen/Vault/Downloads/vvdec-0.2.0.0/source/Lib/CommonLib/UnitTools.h:54,
                 from /home/zen/Vault/Downloads/vvdec-0.2.0.0/source/Lib/CommonLib/UnitTools.cpp:51:
/home/zen/Vault/Downloads/vvdec-0.2.0.0/source/Lib/CommonLib/Mv.h:78:7: note: ‘class Mv’ declared here
   78 | class Mv
      |       ^~
```
seems they used memset when it would probably have been better to use a contructor

--- a/source/Lib/CommonLib/UnitTools.cpp
+++ b/source/Lib/CommonLib/UnitTools.cpp
@@ -1264,7 +1264,8 @@ void PU::getInterMergeCandidates( const PredictionUnit &pu, MergeCtx& mrgCtx, Mo
 
   int r = 0;
   int refcnt = 0;
-  while (uiArrayAddr < maxNumMergeCand)
+  // second condition needed for gcc-10 overflow checking. Required for now. TODO: fix properly
+  while (uiArrayAddr < maxNumMergeCand && uiArrayAddr < MRG_MAX_NUM_CANDS)
   {
     mrgCtx.interDirNeighbours [uiArrayAddr     ] = 1;
     mrgCtx.BcwIdx             [uiArrayAddr     ] = BCW_DEFAULT;
@@ -3085,7 +3086,9 @@ bool PU::getInterMergeSubPuMvpCand(const PredictionUnit &pu, MergeCtx& mrgCtx, b
       for( int x = puPos.x; x < puPos.x + puSize.width; x += puWidth, miLinePtr += g_miScaling.scaleHor( puWidth ) )
       {
         mi.refIdx[0] = mi.refIdx[1] = NOT_VALID;
+        GCC_WARNING_DISABLE_class_memaccess
         memset( mi.mv, 0, sizeof( mi.mv ) );
+        GCC_WARNING_RESET
 
         Position colPos{ x + xOff, y + yOff };
 

--- a/source/Lib/DecoderLib/DecLibRecon.cpp
+++ b/source/Lib/DecoderLib/DecLibRecon.cpp
@@ -516,7 +516,9 @@ bool DecLibRecon::ctuTask( int tid, CtuTaskParam* param )
     for( int ctu = ctuStart; ctu < ctuEnd; ctu++ )
     {
       CtuData& ctuData = cs.getCtuData( ctu, line );
+      GCC_WARNING_DISABLE_class_memaccess
       memset( ctuData.motion, 0, sizeof( CtuData::motion ) );
+      GCC_WARNING_RESET
 
       if( !ctuData.cuPtr[0][0]->slice->isIntra() || cs.sps->getIBCFlag() )
       {

